editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
```

Comment:
At some point we'll also need to consider if blocks can define their own default styles when rendered on the front-end of a site.